core/src/main/java/org/sonarsource/sonarlint/core/util/ProgressWrapper.java

```diff
@@ -55,14 +57,17 @@ public void checkCancel() {
   }
 
   public void setProgress(String msg, float fraction) {
-    handler.setMessage(msg);
+    handler.setMessage(prependPrefix(msg));
     setFraction(fraction);
   }
 
+  private String prependPrefix(String suffix) {
+    return this.msgPrefix != null ? this.msgPrefix + " - " + suffix : suffix;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)